core/domain/collection_services.py

```diff
@@ -312,6 +313,38 @@ def get_multiple_collections_by_id(collection_ids, strict=True):
     return result
 
 
+def get_collection_and_collection_rights_by_id(collection_id):
+    """Returns a tuple for collection domain object and collection rights
+        object.
+
+    Args:
+        collection_id: str. Id of the collection.
+
+    Returns:
+        tuple(collection object or None, collection rights object or None).
+    """
+    collection_and_rights = (
+        datastore_services.fetch_multiple_entities_by_ids_and_models(
+            [
+                ('CollectionModel', [collection_id]),
+                ('CollectionRightsModel', [collection_id])
+            ]))
+
+    collection = None
+    if collection_and_rights[0][0] is not None:
+        collection = get_collection_from_model(
+            collection_and_rights[0][0])
+
+    if collection_and_rights[1][0] is not None:
+        collection_rights = (
+            rights_manager.get_activity_rights_from_model(
+                collection_and_rights[1][0],
+                constants.ACTIVITY_TYPE_COLLECTION))
+    else:
+        collection_rights = None
```

Comment:
You can change this if/else block to the other way too.

Reply:
ok.